content/en/agent/guide/agent-status-page.md

```diff
@@ -0,0 +1,253 @@
+
+### Transactions
+
+This section displays the transactions made by the forwarder, for example:
+```text
+    CheckRunsV1: 2
+    Dropped: 0
+    DroppedOnInput: 0
+    Events: 0
+    HostMetadata: 0
+    IntakeV1: 2
+    Metadata: 0
+    Requeued: 0
+    Retried: 0
+    RetryQueueSize: 0
+    Series: 0
+    ServiceChecks: 0
+    SketchSeries: 0
+    Success: 6
+    TimeseriesV1: 2
+    Errors: 1
+```
+
+Terms and descriptions:
+
+| Term         | Description                                  |
+|--------------|----------------------------------------------|
+| CheckRunsV1  | The number of service check payloads sent.   |
+| IntakeV1     | The number of event payloads sent.           |
+| TimeseriesV1 | The number of metric payloads sent.          |
+| Errors       | The number of errors while sending payloads. |
+
+### API Keys status
+
+This sections shows the status of your configured API key, for example:
+```text
+    API key ending with ab123: API Key valid
+```
+
+## Endpoints
+
+This section displays the list of endpoints in use by the Datadog Agent, for example:
+```text
+  https://app.datadoghq.com - API Key ending with:
+      - ab123
+```
+
+## Logs Agent
+
+If the Logs Agent is enabled, this section displays information on the logs processed and sent, for example:
+```text
+    LogsProcessed: 10
+    LogsSent: 10
+```
+
+## Aggregator
+
+This section displays information on the Agent's aggregator, for example:
+```text
+  Checks Metric Sample: 399
+  Dogstatsd Metric Sample: 123
+  Event: 1
+  Events Flushed: 1
+  Number Of Flushes: 2
+  Series Flushed: 273
+  Service Check: 20
+  Service Checks Flushed: 20
+```
+
+Terms and descriptions:
+
+| Term                    | Description                                                                                                                |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------|
+| Checks Metric Sample    | The total number of metrics sent from the checks to the aggregator.                                                        |
+| Dogstatsd Metric Sample | The total number of metrics sent from the DogStatsD server to the aggregator.                                              |
+| Event                   | The total number of events sent to the aggregator.                                                                         |
+| Service Check           | The total number of service checks sent to the aggregator.                                                                 |
+| Flush                   | The aggregator stores data in a queue. The queue is emptied once per flush interval and the data is sent to the forwarder. |
```

Comment:
It's not a queue, I would say something like: `Number of time aggregated metrics were flush to the forwarder to be sent to DataDog.`